crates/mun_runtime/src/test.rs

```diff
@@ -550,9 +550,9 @@ fn marshal_struct() {
     test_shallow_copy(&mut foo, &foo2, &bool_data, "b");
 
     let mut bar = qux.get::<StructRef>("0").unwrap();
-    let bar2 = qux.get::<StructRef>("0").unwrap();
-    test_shallow_copy(&mut bar, &bar2, &int_data, "0");
-    test_shallow_copy(&mut bar, &bar2, &bool_data, "1");
+    // let bar2 = qux.get::<StructRef>("0").unwrap();
+    // test_shallow_copy(&mut bar, &bar2, &int_data, "0");
+    // test_shallow_copy(&mut bar, &bar2, &bool_data, "1");
```

Comment:
```suggestion
```

Feel free to delete. They'd need to be replaced with tests to check **gc** copies, though.